datasets/mbpp/mbpp_758_unique_sublists.py

```diff
@@ -1,14 +1,14 @@
 def unique_sublists(list1):
 	'''
-	Write a function to count lists within a list. The function should return a dictionary where every list is converted to a tuple and the value of such tuple is the number of its occurencies in the original list.
+	Write a function to count lists within a list. The function should return a dictionary where the value of such list is the number of its occurencies in the original list.
 	'''
 	pass
 
 def check(candidate):
-	assert candidate([[1, 3], [5, 7], [1, 3], [13, 15, 17], [5, 7], [9, 11]] )=={(1, 3): 2, (5, 7): 2, (13, 15, 17): 1, (9, 11): 1}
-	assert candidate([['green', 'orange'], ['black'], ['green', 'orange'], ['white']])=={('green', 'orange'): 2, ('black',): 1, ('white',): 1}
-	assert candidate([[10, 20, 30, 40], [60, 70, 50, 50], [90, 100, 200]])=={(10, 20, 30, 40): 1, (60, 70, 50, 50): 1, (90, 100, 200): 1}
-	assert candidate([['john']])=={('john',): 1}
+	assert candidate([[1, 3], [5, 7], [1, 3], [13, 15, 17], [5, 7], [9, 11]] )=={[1, 3]: 2, [5, 7]: 2, [13, 15, 17]: 1, [9, 11]: 1}
+	assert candidate([['green', 'orange'], ['black'], ['green', 'orange'], ['white']])=={['green', 'orange']: 2, ['black']: 1, ['white']: 1}
+	assert candidate([[10, 20, 30, 40], [60, 70, 50, 50], [90, 100, 200]])=={[10, 20, 30, 40]: 1, [60, 70, 50, 50]: 1, [90, 100, 200]: 1}
+	assert candidate([['john']])=={['john']: 1}
```

Comment:
This test needs to use tuples since this modification isn't valid: list is not a hashable type. Remove? or revert to tuple?